docs/release-notes/0.7/release-0.7.md

```diff
@@ -0,0 +1,142 @@
+          [here](https://www.csie.ntu.edu.tw/~cjlin/talks/recsys.pdf).
+
+* Enabled anomaly detection scenarios
+  ([#1254](https://github.com/dotnet/machinelearning/pull/1254))
+
+    * [Anomaly detection](https://en.wikipedia.org/wiki/Anomaly_detection)
+      enables identifying unusual values or events. It is used in scenarios
+      such as fraud detection (identifying suspicious credit card
+      transactions) and server monitoring (identifying unusual activity). 
+    * This release includes the following anomaly detection techniques:
+      SSAChangePointDetector, SSASpikeDetector, IidChangePointDetector, and
+      IidSpikeDetector. 
+    * Example usage can be found
+      [here](https://github.com/dotnet/machinelearning/blob/7fb76b026d0035d6da4d0b46bd3f2a6e3c0ce3f1/test/Microsoft.ML.TimeSeries.Tests/TimeSeriesDirectApi.cs).
+
+* Enabled using ML.NET in Windows x86 apps
+  ([#1008](https://github.com/dotnet/machinelearning/pull/1008))
+
+    * ML.NET can now be used in x86 apps. 
+    * Some components that are based on external dependencies (e.g.
+      TensorFlow) will not be available in x86. Please open an issue on GitHub
+      for discussion if this blocks you.
+
+* Added the `CustomMappingEstimator` for custom data transformations
+  [#1406](https://github.com/dotnet/machinelearning/pull/1406)
+
+    * ML.NET has a wide variety of data transformations for pre-processing and
+      featurizing data (e.g. processing text, images, categorical features,
+      etc.).
+    * However, there might be application-specific transformations that would
+      be useful to do within an ML.NET pipeline (as opposed to as a
+      pre-processing step). For example, calculating [cosine
+      similarity](https://en.wikipedia.org/wiki/Cosine_similarity) between two
+      text columns (after featurization) or something as simple as creating a
+      new column that adds the values in two other columns.
+    * An example of the `CustomMappingEstimator` can be found
+      [here](https://github.com/dotnet/machinelearning/blob/d68388a1c9994a5b429b194b64b2b0782834cb78/test/Microsoft.ML.Tests/Transformers/CustomMappingTests.cs#L55).
+
+* Consolidated several API concepts in `MLContext`
+  [#1252](https://github.com/dotnet/machinelearning/pull/1252)
+
+    * `MLContext` replaces `LocalEnvironment` and `ConsoleEnvironment` but
+      also includes properties for ML tasks like
+      `BinaryClassification`/`Regression`, various transforms/trainers, and
+      evaluation. More information can be found in
+      [#1098](https://github.com/dotnet/machinelearning/issues/1098).
+    * Example usage can be found
+      [here](https://github.com/dotnet/machinelearning/blob/d68388a1c9994a5b429b194b64b2b0782834cb78/docs/code/MlNetCookBook.md)
+
+* Open sourced [NimbusML](https://github.com/microsoft/nimbusml): experimental
+  Python bindings for ML.NET. 
+
+    * Some teams at Microsoft found it useful to use ML.NET capabilities in
```

Comment:
Would re-word to indicate that NimbusML makes it easy for data scientists to train models in Python and hand them off to .NET developers to include in their apps and services using model scoring in ML.NET

Reply:
Thanks! Addressed.